### DISCUSSION TITLE
java.net.SocketTimeoutException:Connect timed out

### DISCUSSION THREAD

**Discussion by arshad95** [dbeaver/dbeaver#19028] [Q&A] [answered]
### Description

Hi,

When i try to open my ORACLE DB i have this message : 

<img width="283" alt="image" src="https://user-images.githubusercontent.com/29165642/217852667-897b3906-47e7-4d7e-93d8-88520e8a9ad5.png">

When i was in DBeaver 7.2.1 it's working fine, now i'm on 22.3.4. Did you know can i fix this issue please ? 

And when i close the pop-up i have another one with this message : 

<img width="547" alt="image" src="https://user-images.githubusercontent.com/29165642/217853208-d145f184-c433-45b7-86c1-53c80dd2c21a.png">

But i have correctly upload the oracle.jdbc.OrcleDriver :

<img width="417" alt="image" src="https://user-images.githubusercontent.com/29165642/217853812-0f0f2668-58de-458b-babd-503391bee04c.png">

### DBeaver Version

22.3.4

### Database and driver

Database - Oracle

**Comment by E1izabeth** [comment, marked as answer]
Try deleting all blue items in the list Libraries tab.

**Comment by E1izabeth** [comment]
Please let me know if it helps.

**Comment by arshad95** [comment]
Hi E1izabeth,

Yes it's working thank you

**Reply by divyanshu122** [reply]
hello i am connecting postgresql in dbeaver i am getting error
![Screenshot (1)](https://github.com/dbeaver/dbeaver/assets/134134679/95b12503-e543-4797-9365-4fc831abc091)

**Reply by divyanshu122** [reply]
after clicking ok getting this error
![Screenshot (2)](https://github.com/dbeaver/dbeaver/assets/134134679/5b69af4c-d393-4ca6-8c30-3c6f94763d5a)

**Comment by divyanshu122** [comment]
after clicking ok i am getting this error
![Uploading Screenshot (2).png…]()

**Reply by E1izabeth** [reply]
Have you tried the solution proposed?

**Reply by divyanshu122** [reply]
yes i have tried but not resolved giving timeout
plz help me

**Reply by E1izabeth** [reply]
Are you getting a timeout while trying to download driver files? If yes, it must be your network connection issue.

**Reply by divyanshu122** [reply]
i am getting this 
![Screenshot (3)](https://github.com/dbeaver/dbeaver/assets/134134679/96d38bb3-02a5-422b-9439-98e2c83ec67f)
![Screenshot (4)](https://github.com/dbeaver/dbeaver/assets/134134679/2b8f6470-0f26-4192-b5c8-fa13eb2080b5)

**Reply by divyanshu122** [reply]
after restart dbeaver i got connection time out

**Reply by divyanshu122** [reply]
can help what can be the connection issue

**Reply by E1izabeth** [reply]
On some of your screenshots I see such icon, which means that you have no internet connection. Can it be a reason?
![image](https://github.com/dbeaver/dbeaver/assets/28875055/dda77d73-29bd-416f-8bf8-3f229c546600)

**Reply by divyanshu122** [reply]
no that was not the reason i am connected with proper internet connection

**Reply by E1izabeth** [reply]
Do you use SSL?

**Reply by divyanshu122** [reply]
no
